app-text/poppler/poppler-0.63.0.recipe

```diff
@@ -0,0 +1,433 @@
+	devel:libtiff$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+
+# GLIB
+SUMMARY_glib="$SUMMARY (glib bindings)"
+DESCRIPTION_glib="$DESCRIPTION (glib bindings)"
+
+PROVIDES_glib="
+	poppler${secondaryArchSuffix}_glib = $portVersion
+	lib:libpoppler${secondaryArchSuffix}_glib = $libGlibVersionCompat
+	"
+REQUIRES_glib="
+	haiku$secondaryArchSuffix
+	poppler$secondaryArchSuffix == $portVersion base
+	lib:libcairo$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libfontconfig$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libgio_2.0$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgobject_2.0$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:liblcms2$secondaryArchSuffix
+	lib:libopenjp2$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libtiff$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+# QT5
+SUMMARY_qt5="$SUMMARY (Qt5 bindings)"
+DESCRIPTION_qt5="$DESCRIPTION (Qt5 bindings)"
+
+PROVIDES_qt5="
+	poppler${secondaryArchSuffix}_qt5 = $portVersion
+	lib:libpoppler${secondaryArchSuffix}_qt5 = $libQt5VersionCompat
+	"
+REQUIRES_qt5="
+	haiku$secondaryArchSuffix
+	poppler$secondaryArchSuffix == $portVersion base
+	lib:libcurl$secondaryArchSuffix
+	lib:libfontconfig$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:liblcms2$secondaryArchSuffix
+	lib:libopenjp2$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	lib:libQt5Xml$secondaryArchSuffix
+	lib:libtiff$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libcairo$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libfontconfig$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libgio_2.0$secondaryArchSuffix
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libgobject_2.0$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:liblcms2$secondaryArchSuffix
+	#devel:libnss3$secondaryArchSuffix
+	devel:libopenjp2$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	devel:libQt5Xml$secondaryArchSuffix
+	devel:libtiff$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gawk
+	cmd:grep
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:perl
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:python2
+	"
+
+BUILD()
+{
+	cmake . $cmakeDirArgs \
+		-DCMAKE_CXX_FLAGS="-D_BSD_SOURCE" \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DENABLE_XPDF_HEADERS=ON
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	prepareInstalledDevelLibs libpoppler \
```

Comment:
The current style would be perfect for a `defineDebugInfoPackage` or for a `packageEntries`, but for a `prepareInstalledDevelLibs` I'd recommend:
```diff
-	prepareInstalledDevelLibs libpoppler \
+	prepareInstalledDevelLibs \
+		libpoppler \
 		libpoppler-cpp \
 		libpoppler-glib \
 		libpoppler-qt5
```
(Imo this style variant is slitghly more appropriate, but I'm OK with the current style.)

Reply:
Sorry, I had not noticed that the style for `prepareInstalledDevelLibs` was already in `poppler-0.62.0.recipe`. So there is no need to change this.